### PR TITLE
Document support bundle upload and new metrics

### PR DESCRIPTION
**Pull request by lfitchett**
This also removes the experimental flags.

Note I didn't rename built-in-logs-pull.md since some documentation links to it: https://docs.microsoft.com/en-us/azure/iot-edge/how-to-edgeagent-direct-method#experimental-methods. We should probably update this as well.

### CODE REVIEW

File: doc/BuiltInMetrics.md [hunk 1]
```diff
@@ -3,7 +3,12 @@
 The IoT Edge Hub and Edge Agent module expose a number of metrics in the Prometheus exposition format that provide insights into its operational state.
 
 ## How to enable
+### Version 1.0.10+
+As of release 1.0.10, metrics are exposed automatically exposed on http port **9600** of the Edge Hub and Edge Agent module.
```
Comment (CindyXing) [inline doc/BuiltInMetrics.md:7]: nit pick... 

exposed by default on http port ...

Comment (damonbarry) [reply doc/BuiltInMetrics.md:7]: +1

File: doc/BuiltInMetrics.md [hunk 2]
```diff
@@ -4,31 +4,10 @@ The IoT Edge Hub and Edge Agent module expose a number of metrics in the Prometh
 
 ## How to enable
 ### Version 1.0.10+
-As of release 1.0.10, metrics are exposed automatically exposed on http port **9600** of the Edge Hub and Edge Agent module. 
+As of release 1.0.10, metrics are exposed automatically exposed by default on http port **9600** of the Edge Hub and Edge Agent module.
```
Comment (damonbarry) [inline doc/BuiltInMetrics.md:7]: You have "exposed automatically exposed"

Comment (lfitchett) [reply doc/BuiltInMetrics.md]: oops :)

**Pull request merged**